### PR TITLE
Create PROJECT_LIFECYCLE.md

### PR DESCRIPTION
**Pull request by caniszczyk**
RFC @ucfoundation/tac 
Issue #3

### CODE REVIEW

File: PROJECT_LIFECYCLE.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+* preferred maturity level (see stages below)
+* list of project's official communication channels (slack, irc, mailing lists)
+* link to project's website 
+* links to social media accounts
+* existing financial sponsorship
+* infrastructure needs or requests 
+
+### Project Acceptance Process
+* Projects are required to present their proposal at a TAC meeting
+* The TAC may ask for changes to bring the project into better alignment with the OpenJS Foundation (adding a governance document to a repository or adopting a Code of Conduct, for example).
```
Comment (Mec-iS) [inline PROJECT_LIFECYCLE.md]: ~OpenJS~ typo?

**Comment by sarob** [comment]
To better understand, I am using Kepler.gl as a reference, since Uber plans on contributing the project to UCF. Can I assume that Kepler.gl is at the growth stage? The project has been an active open source project over some time. Do you think some projects like Kepler.gl could start at the impact stage?

File: PROJECT_LIFECYCLE.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Projects must be formally proposed via GitHub. Project proposals submitted to th
 
 ### Project Acceptance Process
 * Projects are required to present their proposal at a TAC meeting
-* The TAC may ask for changes to bring the project into better alignment with the OpenJS Foundation (adding a governance document to a repository or adopting a Code of Conduct, for example).
+* The TAC may ask for changes to bring the project into better alignment with the Urban Computing Foundation (adding a governance document to a repository or adopting a Code of Conduct, for example).
```
Comment (danlopez00) [inline PROJECT_LIFECYCLE.md:39]: Looks good to me

**Comment by bhenhsi** [comment]
> To better understand, I am using Kepler.gl as a reference, since Uber plans on contributing the project to UCF. Can I assume that Kepler.gl is at the growth stage? The project has been an active open source project over some time. Do you think some projects like Kepler.gl could start at the impact stage?

@sarob
A project can propose its preferred maturity level in its submission proposal, ref section II.  A project can propose to enter at the "Impact" stage if it believes its maturity meet what's outlined in this proposal. The TAC should/will review its proposal and maturity level fairly.

**Pull request merged**